modules/social_features/social_profile/config/install/core.entity_view_display.profile.profile.small_teaser.yml

```diff
@@ -3,6 +3,9 @@ status: true
 dependencies:
   config:
     - core.entity_view_mode.profile.small_teaser
+    - field.field.profile.profile.field_birthdate
```

Comment:
@agami4 Please remove all those new fields, since they are not in a Distro by default.
Leave only changes that you made on purpose.